lang/system/pl.xml

```diff
@@ -308,44 +556,268 @@ System Vita3K jest gotowy!</completed_setup>
 				<mm_dd_yyyy>MM/DD/RRRR</mm_dd_yyyy>
 			</date_format>
 			<time_format name="Format godziny">
-				<clock_12_hour>Zegar 12-godz</clock_12_hour>
-				<clock_24_hour>Zegar 24-godz</clock_24_hour>
+				<clock_12_hour>Zegar 12-godz.</clock_12_hour>
+				<clock_24_hour>Zegar 24-godz.</clock_24_hour>
 			</time_format>
 		</date_time>
 		<language name="Język">
 			<system_language>Język systemu</system_language>
-			<input_language name="język wprowadzania">
-				<keyboards name="Klawiatury">
+			<input_language name="Języki wprowadzania">
+				<keyboards name="Rodzaj klawiatury">
 					<ime_languages>
-						<lang id="1">Duński</lang>
-						<lang id="2">Niemiecki</lang>
-						<lang id="262144">Angielski (Wielka Brytania)</lang>
-						<lang id="4">Angielski (Stany Zjednoczone)</lang>
-						<lang id="8">Hiszpański</lang>
-						<lang id="16">Francuski</lang>
-						<lang id="32">Włoski</lang>
-						<lang id="64">Niderlandzki</lang>
-						<lang id="128">Norweski</lang>
+						<lang id="1">Duńska</lang>
+						<lang id="2">Niemiecka</lang>
+						<lang id="262144">Angielska (Wielka Brytania)</lang>
+						<lang id="4">Angielska (Stany Zjednoczone)</lang>
+						<lang id="8">Hiszpańska</lang>
+						<lang id="16">Francuska</lang>
+						<lang id="32">Włoska</lang>
+						<lang id="64">Niderlandzka</lang>
+						<lang id="128">Norweska</lang>
 						<lang id="256">Polska</lang>
-						<lang id="131072">Portugalski (Brazylia)</lang>
-						<lang id="512">Portugalski (Portugalia)</lang>
-						<lang id="1024">Rosyjski</lang>
-						<lang id="2048">Fiński</lang>
-						<lang id="4096">Szwedzki</lang>
-						<lang id="524288">Turkish</lang>
+						<lang id="131072">Portugalska (Brazylia)</lang>
+						<lang id="512">Portugalska (Portugalia)</lang>
+						<lang id="1024">Rosyjska</lang>
+						<lang id="2048">Fińska</lang>
+						<lang id="4096">Szwedzka</lang>
+						<lang id="524288">Turecka</lang>
```

Comment:
```suggestion
						<lang id="524288">Turecki</lang>
```

Reply:
Revert all other, is already good

Reply:
Done, I was confused as Polish was 'Poland' other not really in that form.